Automattic/newspack-newsletters
# Set up beta releases

semantic-release config tweak for [pre-release](https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#pre-release-branches)

## How to test this

1. Change `package.json`'s `release.branches` key to 
```
    "branches": [
      "release",
      {
        "name": "add/setup-alpha-releases",
        "prerelease": "alpha"
      }
    ],
```
2. Do a dry run: 
```
GH_TOKEN=<YOUR TOKEN GOES HERE> npm run semantic-release -- --dry-run --no-ci
```

3. Observe `Release note for version 1.0.0-alpha.1` in the output, followed by release notes.
